winterfell/forms/VBA-21-0966-ARE.json

```diff
@@ -1,4 +1,8 @@
 {
+  "unofficialTitle": "Intent to File",
```

Comment:
@qwwqwwq am I allowed to do this? I just hate the official name and description pulled from the form. I think if we want things to be more presentable, we'll have to redefine some fields.
But, since this file is auto-generated, is there a better place to add fields?
We may have to look into a pipeline where we combine autogenerated stuff with additional fields we define to create this file.